tests/e2e/util.go

```diff
@@ -5570,3 +5570,27 @@ func waitForPvcToBeDeleted(ctx context.Context, client clientset.Interface, pvcN
 	framework.Logf("Status of pvc is: %v", pvc.Status.Phase)
 	return waitErr
 }
+
+/* This util method fetches pvc events list and matches the events list with the
+specified error message and returns true if expected error found */
+func getEventsListAndVerifyPvcError(client clientset.Interface, namespace string,
```

Comment:
may be change it to `wait4EventWithReason`

also instead of accepting PVC, we can accept the object name(string), thus making this routine generic

Reply:
done